development/hotfix.md

```diff
@@ -0,0 +1,31 @@
+### What
+
+A hotfix is required when a critical bug or security issue is discovered in a public release which is deemed to be too urgent to resolve by using the normal release cadence.
+
+### When
+
+In most cases, they will only be applied to the currently active release, and we'll encourage users to upgrade to newer releases in the event a critical issue is found in an old version of the software. However, consideration will be made for bugs or security issues which severely impact the usage of old versions.
+
+In the event that a bug or security issue is active in the current release, the product owner and the development team lead will make the call to release a hotfix.
+
+### How
+
+The process for creating a hotfix release is the following. In this example scenario, a hotfix is being applied to the 1.14.0 version of a project.
+
+1. When an issue is deemed crticial enough to warrant a hotfix, the "showstopper" tag is immediately applied to it. The issue immediately becomes the teams highest priority.
+2. The tag for 1.14.0 is checked out.
+3. A branch is created on origin based on the tag for the original release. It should be in the form of "hotfix-for-[previous-release]". In the case of a hotfix based on 1.14.0, the branch would be named "hotfix-for-1.14.0"
+4. The bug or security issue is fixed and applied to the hotfix-for-1.14.0 branch that exists on origin.
+5. A new release is created using [our standard process](product_release.md)
```

Comment:
Does it make sense to mention that the release should increment the version to 1.14.1 in this example?